client/components/Main.js

```diff
@@ -28,6 +29,7 @@ class Main extends React.Component {
           <Switch>
             <Route exact path="/" component={Home}></Route>
             <Route exact path="/products" component={AllProducts} />
+            <Route exact path="/countries" component={CrearteCountry} />
```

Comment:
let's change it to manage-countries for consistency

Reply:
Done!